### PR TITLE
Update customer-key-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Teams to the list as Customer Key support for Microsoft Teams now in Public Preview! ( https://techcommunity.microsoft.com/t5/microsoft-teams-blog/customer-key-support-for-microsoft-teams-now-in-public-preview/ba-p/1999893) 

Reference Doc:- https://docs.microsoft.com/en-us/microsoft-365/compliance/customer-key-tenant-level

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4753

File: microsoft-365/compliance/customer-key-overview.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Customer Key only encrypts data at rest in the cloud. Customer Key does not work
 
 A data encryption policy defines the encryption hierarchy to encrypt data using each of the keys you provide as well as the availability key protected by Microsoft. You create DEPs using PowerShell cmdlets, which are different for each service, and assign those DEPs to encrypt application data. For example:
 
-**Exchange Online and Skype for Business** You can create up to 50 DEPs per tenant. You associate DEPs to your Customer Keys in Azure Key Vault and then assign DEPs to individual mailboxes. When you assign a DEP to a mailbox:
+**Exchange Online, Teams and Skype for Business** You can create up to 50 DEPs per tenant. You associate DEPs to your Customer Keys in Azure Key Vault and then assign DEPs to individual mailboxes. When you assign a DEP to a mailbox:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md]:
```suggestion
**Exchange Online, Teams, and Skype for Business** You can create up to 50 DEPs per tenant. You associate DEPs to your Customer Keys in Azure Key Vault and then assign DEPs to individual mailboxes. When you assign a DEP to a mailbox:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Comment by tiburd** [comment]
This change has been incorporated into the updated article via a different PR. Closing this PR.

**Pull request closed without merging**